### PR TITLE
V5 ga 53 security policies editor

### PR DESCRIPTION
**Pull request by ljubebiza**

### CODE REVIEW

File: src/store/UserStore.ts [hunk 1]
```diff
@@ -110,7 +110,7 @@ export const useUserStore = defineStore('user', () => {
       return accessLevels.includes(group)
     })
     const devMode = process.env.NODE_ENV === 'development'
-    return !!userHasAccessLevel || !!devMode
+    return !!userHasAccessLevel || !!devMode || true
```
Comment (Aviv-Galmidi) [inline src/store/UserStore.ts:113]: Remove from PR

File: vue.config.js [hunk 2]
```diff
@@ -12,7 +12,7 @@ module.exports = {
       },
       '^/reblaze/api': {
         pathRewrite: {'^/reblaze/api': '/api'},
-        target: 'http://127.0.0.1:8000',
+        target: 'http://34.107.111.154:8911',
```
Comment (Aviv-Galmidi) [inline vue.config.js:15]: Remove from PR

File: src/views/security/SecurityPoliciesEditor.vue [hunk 3]
```diff
@@ -41,74 +41,65 @@
...
-                           @change="emitDocUpdate"
-                           @input="validateInput($event, isSelectedDomainMatchValid)"
-                           v-model="localDoc.match"
-                           :disabled="localDoc.id === '__default__'"
-                           :readonly="localDoc.id === '__default__'"
-                           title="Enter a regex to match hosts headers (domain names)">
-                    <span class="icon is-small is-left has-text-grey">
-                      <i class="fas fa-code"></i>
-                    </span>
+                    <form-input data-qa="security-policies-match-input"
```
Comment (Aviv-Galmidi) [inline src/views/security/SecurityPoliciesEditor.vue:61]: Match Host doesn't show data from the Security Policy



https://user-images.githubusercontent.com/10487148/235424385-59caf258-ef8a-4cee-98ef-7276a144370e.mov

File: src/views/security/SecurityPoliciesEditor.vue [hunk 4]
```diff
@@ -41,74 +41,65 @@
...
+                  <label class="is-small label-default-style">Tags</label>
                   <div class="control"
                        data-qa="tag-input">
                     <tag-autocomplete-input :initial-tag="selectedDocTags"
                                             selection-type="multiple"
                                             @tag-changed="selectedDocTags = $event"/>
-                    <labeled-tags title="Automatic Tag"
-                                  :tags="automaticTags"/>
+                    <label class="label-default-style">Automatic Tag</label>
+                    <custom-tag v-for="(tag, index) in automaticTags"
```
Comment (Aviv-Galmidi) [inline src/views/security/SecurityPoliciesEditor.vue:82]: Automatic tags flicker on hover




https://user-images.githubusercontent.com/10487148/235424447-86d57298-ff66-4de6-91f7-beadffd2f252.mov

File: src/views/security/SecurityPoliciesEditor.vue [hunk 5]
```diff
@@ -374,63 +356,49 @@
...
+                              id="Acitve-mode-acl"
+                              label="Active mode"
+                              @change="emitDocUpdate"
+                              data-qa="acl-active-checkbox"
+                              class="current-entry-acl-active"
+                              :checked="mapEntry.acl_active"
+                              v-model:checkedValue="mapEntry.acl_active"/>
                         </div>
                         <hr/>
                         <div class="field">
```
Comment (Aviv-Galmidi) [inline src/views/security/SecurityPoliciesEditor.vue:404]: `Fork path mapping` button was not converted

File: src/views/security/SecurityPoliciesEditor.vue [hunk 6]
```diff
@@ -218,45 +209,41 @@
...
                           </div>
                         </div>
                         <div class="field">
-                          <label class="label is-small">
-                            Match Path
-                          </label>
                           <div class="control has-icons-left">
-                            <input v-model="mapEntry.match"
-                                   class="input is-small current-entry-match"
+                            <form-input name="mapEntryMatch"
```
Comment (Aviv-Galmidi) [inline src/views/security/SecurityPoliciesEditor.vue:233]: This input does not contain any data on startup

Screen Shot 2023-05-01 at 10 39 35

File: src/views/security/SecurityPoliciesEditor.vue [hunk 7]
```diff
@@ -445,13 +413,13 @@
...
-                                  class="button is-small is-pulled-right is-danger is-light remove-entry-button"
-                                  @click="removeMapEntry(mapIndex)"
-                                  v-if="!isProtectedEntry(mapEntry)">
-                            Delete
-                          </button>
+                          <form-button label="Delete"
+                            class="is-pulled-right"
+                            title="Delete this profile"
+                            data-qa="delete-location-btn"
+                            variant="tertiary"
```
Comment (Aviv-Galmidi) [inline src/views/security/SecurityPoliciesEditor.vue:420]: Should be negative

**Review by Aviv-Galmidi — CHANGES_REQUESTED** [review]
Tests need to pass

File: src/views/security/SecurityPoliciesEditor.vue (hunk 3 above)
Comment (SrdjanReblaze) [reply src/views/security/SecurityPoliciesEditor.vue:61]: done

File: src/views/security/SecurityPoliciesEditor.vue (hunk 6 above)
Comment (SrdjanReblaze) [reply src/views/security/SecurityPoliciesEditor.vue:233]: done

File: src/views/security/SecurityPoliciesEditor.vue (hunk 7 above)
Comment (SrdjanReblaze) [reply src/views/security/SecurityPoliciesEditor.vue]: done

File: src/views/security/SecurityPoliciesEditor.vue (hunk 5 above)
Comment (SrdjanReblaze) [reply src/views/security/SecurityPoliciesEditor.vue:405]: done

**Review by Aviv-Galmidi — CHANGES_REQUESTED** [review]
1. Delete button is messed up
Screen Shot 2023-05-18 at 13 05 04

2. Button icon is missing
Screen Shot 2023-05-18 at 13 08 07

3. Table column widths changed drastically
Old
Screen Shot 2023-05-18 at 13 08 51
New
Screen Shot 2023-05-18 at 13 08 46

**Comment by SrdjanReblaze** [comment]
> 1. Delete button is messed up
>    Screen Shot 2023-05-18 at 13 05 04
> 2. Button icon is missing
>    Screen Shot 2023-05-18 at 13 08 07
> 3. Table column widths changed drastically
>    Old
>    Screen Shot 2023-05-18 at 13 08 51
>    New
>    Screen Shot 2023-05-18 at 13 08 46

done

**Review by Aviv-Galmidi — DISMISSED** [review]

**Review by Aviv-Galmidi — APPROVED** [review]

**Review by Aviv-Galmidi — APPROVED** [review]

**Pull request merged**